docs/about/application-setup/setup-using-nix/README.md

```diff
@@ -0,0 +1,13 @@
+---
+id: Setup Using Nix
+sidebar_position: 1
```

Comment:
```suggestion
sidebar_position: 3
```

Let's be explicit that the Nix guide is after the Manual guide here. The should not both have a `sidebar_position` of 1.